<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EIrepack v1.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Папка с игрой (содержит game.exe):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="src_folder"/>
       </item>
       <item>
        <widget class="QPushButton" name="src_folder_choser">
         <property name="text">
          <string>Обзор...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="in_place">
       <property name="text">
        <string>Распаковка &quot;На месте&quot; (использовать с осторожностью!)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Папка для распаковки (здесь будут результаты работы):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="dst_folder"/>
       </item>
       <item>
        <widget class="QPushButton" name="dst_folder_choser">
         <property name="text">
          <string>Обзор...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="need_copy">
       <property name="text">
        <string>Копировать файлы в рабочую папку</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="need_extracting">
       <property name="text">
        <string>Распаковывать архивы</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="need_converting">
       <property name="text">
        <string>Конвертировать ресурсы</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="need_text_merging">
       <property name="text">
        <string>Объединить игровые тексты</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="need_verbose">
       <property name="text">
        <string>Показать подробную историю действий</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>Начать распаковку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_step">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progress_all">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
